app/components/Texting/components/FormView/Form.module.scss

```diff
@@ -27,10 +27,38 @@
   }
 }
 
+.phoneInputBox {
+  display: flex;
+  margin-top: 10px;
+  background-color: $color-white;
+  border: 1px solid #DDDDDD;
+  border-radius: 2px;
+  letter-spacing: 1px;
+  &:active, &:hover, &:focus {
+    box-shadow: 0 2px 4px 0 rgba(0,0,0,0.1);
+  }
+}
+
+.phoneInputPrefix {
+  padding: 0.9375rem 0 0.9375rem 0.9375rem;
+  color: $color-grey6;
+  font-size: 1.0625rem;
+}
+
+.phoneInput {
+  border: none;
+  background: $color-white;
+  padding-left: 5px;
+  margin: 0; /* to avoid Safari for adding extra margin */
+  box-shadow: none !important;
```

Comment:
Please avoid using `!important` when unnecessary, since it's a large hammer to wield, and it's difficult to override the effects of an `!important` later on.

I'm assuming you wrote this because you're running into an issue with [CSS specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity). More specifically, I'm assuming that the `:hover` pseudo-class style on the base `input` element is causing its `box-shadow` property to take priority over your non-pseudo-class `.phoneInput` selector here.

To achieve what you want in a safer way, all you need to do is escalate the specificity of your selector to be greater than that of the `input:hover` selector. Tag selectors have the lowest priority, followed by class and pseudo-class selectors, and then followed by ID selectors. If you have multiple selectors of the same level, then they take priority over ones with fewer selectors.

For example, `input` has less specificity than `.phoneInput`, since `input` is a tag selector and `.phoneInput` is a class selector. `input:hover` has higher specificity than `.phoneInput`, though, because `input:hover` has one level 2 selector (`:hover`) and one level 1 selector (`input`) while `.phoneInput` only has one level 2 selector (`.phoneInput`).

However, `.phoneInput:hover` would win over `input:hover` because `.phoneInput:hover` has _two_ level 2 selectors (`.phoneInput` and `:hover`).

If you just override the `box-shadow` style within a pseudo-class selector of the `.phoneInput`, then you'll be able to successfully take priority over the `input:hover` selector. Note that our [actual CSS style for setting the `box-shadow` on input elements targets three pseudo-classes](https://github.com/ShelterTechSF/askdarcel-web/blob/7e4103e80cedfd96566c15f0800eac9913c86fc0/app/styles/st-base/_forms.scss#L20-L22) — `:hover`, `:active`, and `:focus` — so you'll have to override all three.